distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/ListPluginsCommandTests.java

```diff
@@ -221,6 +265,36 @@ public void testPluginWithVerboseMultiplePlugins() throws Exception {
                 terminal.getOutput());
     }
 
+    public void testPluginWithVerboseUberPlugins() throws Exception {
+        buildFakeUberPlugin(env, "fake uber desc", "uber_plugin", "fake_plugin1,fake_plugin2");
+        buildFakePlugin(env, "uber_plugin", "fake desc 1", "fake_plugin1", "org.fake");
+        buildFakePlugin(env, "uber_plugin",  "fake desc 2", "fake_plugin2", "org.fake2");
+        String[] params = { "-v" };
+        MockTerminal terminal = listPlugins(home, params);
+        assertEquals(
+            buildMultiline(
+                "Plugins directory: " + env.pluginsFile(),
+                "uber_plugin:fake_plugin1",
+                "- Plugin information:",
+                "Uber Plugin: uber_plugin",
```

Comment:
It seems silly to print out the meta plugin this is part of, given it is already grouped under the the meta plugin.